app/src/main/java/com/example/nidoqueue/controller/ExperimentManager.java

```diff
@@ -49,42 +48,42 @@ public DataCalc getCurrentCalc() {
-                                            ownerId.put("owner", android_id);
-                                            db.collection("experiments")
-                                                    .document(exp.getName().toLowerCase())
-                                                    .set(ownerId, SetOptions.merge());
-
-                                            db.collection("users")
-                                                    .document(android_id)
-                                                    .update("createdExp", FieldValue.arrayUnion(exp));
-                                        }
-                                    });
-                        }
-                    } else {
-                        Log.d("FireStore", "Failed with: ", task.getException());
-                    }
-                });
+//        FirebaseFirestore db = contextManager.getActivity().getDB();
+//        User user = userControl.getUser();
+//
+//        Log.d("Test", "Clicked");
+//
+//        db.collection("experiments")
+//                .document(exp.getName().toLowerCase())
+//                .get()
+//                .addOnCompleteListener(task -> {
+//                    if (task.isSuccessful()) {
+//                        DocumentSnapshot document = task.getResult();
+//                        if (document.exists()) {
+//                            Log.d("FireStore", "Document exists!");
+//                            Toast.makeText(contextManager.getActivity().getApplicationContext(), "Name already exists!\nTry use other name", Toast.LENGTH_LONG).show();
+//                        } else {
+//                            user.createExp(exp);
+//                            db.collection("experiments")
+//                                    .document(exp.getName().toLowerCase())
+//                                    .set(exp)
+//                                    .addOnCompleteListener(task1 -> {
+//                                        if (task1.isSuccessful()) {
+//                                            Map<String, String> ownerId = new HashMap<>();
+//                                            ownerId.put("owner", android_id);
+//                                            db.collection("experiments")
+//                                                    .document(exp.getName().toLowerCase())
+//                                                    .set(ownerId, SetOptions.merge());
+//
+//                                            db.collection("users")
+//                                                    .document(android_id)
+//                                                    .update("createdExp", FieldValue.arrayUnion(exp));
+//                                        }
+//                                    });
+//                        }
+//                    } else {
+//                        Log.d("FireStore", "Failed with: ", task.getException());
+//                    }
+//                });
```

Comment:
Please delete this code instead of commenting it out, that way no one has to go back and delete it later.